src/server/info_collector.cpp

```diff
@@ -304,7 +304,8 @@ void info_collector::on_storage_size_stat(int remaining_retry_count)
 hotspot_calculator *info_collector::get_hotspot_calculator(const std::string &app_name,
                                                            const int partition_num)
 {
-    auto iter = _hotspot_calculator_store.find(app_name);
+    std::string app_name_count = app_name + std::to_string(partition_num);
+    auto iter = _hotspot_calculator_store.find(app_name_count);
```

Comment:
You may need add comment, otherwise the `key` is set `app_name + partition_num` may confuse others when read the code.